vtex-apps/my-account
# Feature/idcs 833/remove react vtexid

#### What did you change? \*

Replace client usage for direct call to `/logout` route; remove `vtex.react-vtexid` dependency from project
Also, this removes unused arab messages.

#### How to test it? \*

You can test it at [this workspace](https://recorrenciaqa.vtexcommercestable.com.br/_secure/Account/Orders/?workspace=rafaprtest#/orders). Click the MyAccount logout button and check that the logout route is correctly called with an absolute `returnUrl` value. You will see that the redirects correctly occur to the home page, taking `rootPath` into consideration.

![image](https://user-images.githubusercontent.com/22064061/142264640-7e5a6679-07c5-45f7-a10c-e4a943992c73.png)

#### Types of changes \*
